src/zui/ZUIMapControls.tsx

```diff
@@ -57,7 +64,7 @@ const ZUIMapControls: React.FC<Props> = ({
         if (isAwaitingInitialPosition) {
           setIsAwaitingInitialPosition(false);
         }
-        if (isUserLocationVisibleRef.current) {
+        if (isUserLocationVisibleRef.current && autoUpdateRef.current) {
```

Comment:
If I understand correctly how this works, every map that uses `ZUIMapControls` is always watching the position using `navigator.geolocation.watchPosition()` after the user has clicked it once. But maps with `autoUpdate == false` just don't do anything about it.

To me this seems a little like an unnecessary way of doing it. Why should every map listen for position changes but not actually do anything about it?

I believe (but correct me if I'm wrong or if I'm missing something) that an alternative solution that would avoid this, would be to start/stop listening if `autoUpdate` changes. And if `autoUpdate == false`, clicking the button could just not start watching.